contract_manager/scripts/generate_governance_set_fee_payload.ts

```diff
@@ -0,0 +1,41 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { DefaultStore } from "../src";
+import { Chain } from "../src/chains";
+
+const parser = yargs(hideBin(process.argv))
+  .usage("Usage: $0 --chain <chain_id> --fee <fee> --exponent <exponent>")
+  .options({
+    chain: {
+      type: "string",
+      demandOption: true,
+      desc: "Chain for which to generate the Set Fee payload",
+    },
+    fee: {
+      type: "number",
+      demandOption: true,
+      desc: "The new fee to set",
+    },
+    exponent: {
+      type: "number",
+      demandOption: true,
+      desc: "The new fee exponent to set",
+    },
+  });
+
+async function main() {
+  const argv = await parser.argv;
+  const chain = argv.chain;
+  const fee = argv.fee;
+  const exponent = argv.exponent;
+
+  const chain_obj: Chain = DefaultStore.chains[chain];
```

Comment:
Not sure why you aded `: Chain`.
Probably we should check that this chain exists and throw some better error if it does not